native/core/src/execution/datafusion/planner.rs

```diff
@@ -2388,7 +2388,7 @@ mod tests {
         };
 
         let expr = spark_expression::Expr {
-            expr_struct: Some(Eq(Box::new(spark_expression::Equal {
+            expr_struct: Some(Eq(Box::new(spark_expression::BinaryExpr {
```

Comment:
I'm guessing this is the only place where the expression needed to be updated explicitly?

Reply:
Yes